exercises/nucleotide-count/tests/nucleotide-count.rs

```diff
@@ -17,19 +17,31 @@ fn check_dna(s: &str, pairs: &[(char, usize)]) {
 
 #[test]
 fn test_count_empty() {
-    assert_eq!(dna::count('A', ""), 0);
+    assert_eq!(dna::count('A', "").unwrap(), 0);
```

Comment:
Agreed. For implementations that expect Result or Option, I usually start with some `is_err` and `is_ok` tests just to make it clear.